src/process-analysis/process-analysis.bib

```diff
@@ -0,0 +1,15 @@
+@report{P1,
+    author = {Kristiyan Mariyan Georgiev and Mads Heilmann and Martin Kedmenec and Sebastian Hemmer Bech Mygind and
+ Hasnain Ali Sajad and Jawad Mehmood Khan Qayyum and Simon Woidemann},
```

Comment:
!

Reply:
Is it the indentation that is wrong here?

Reply:
What names are wrong?

Reply:
```suggestion
and Simon Woidemann},
```

Reply:
They were authors on the P1 project though